plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/AbstractTestDeltaLakeCreateTableStatistics.java

```diff
@@ -460,6 +461,7 @@ public void testMultiFileTable()
     }
 
     @Test
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/11407", match = "to be greater than")
```

Comment:
Please undo. I am sorry by `annotate` I meant to create github issues. We should not add `@Flaky` annotation blindly.